bedrock/firefox/templates/firefox/hello/start-45.html

```diff
@@ -0,0 +1,39 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
```

Comment:
Can we use a different lang file? I don't want to create a folder for such a short-lived page
```
{% add_lang_files "firefox/hello-start-45" %}
```

Double checked locally to verify that production would redirect correctly.